x-pack/plugins/ecs_data_quality_dashboard/server/lib/data_stream/results_data_stream.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { InstallParams } from '@kbn/data-stream-adapter';
+import { DataStreamSpacesAdapter } from '@kbn/data-stream-adapter';
+import { ecsFieldMap } from '@kbn/data-stream-adapter/ecs';
+import { resultsFieldMap } from './results_field_map';
+
+const TOTAL_FIELDS_LIMIT = 2500;
+
+export const RESULTS_DATA_STREAM_NAME = '.kibana-dqa-dashboard-results';
+
+const RESULTS_INDEX_TEMPLATE_NAME = '.kibana-dqa-dashboard-results-index-template';
+const RESULTS_COMPONENT_TEMPLATE_NAME = '.kibana-dqa-dashboard-results-mappings';
+const ECS_COMPONENT_TEMPLATE_NAME = '.kibana-dqa-dashboard-ecs-mappings';
```

Comment:
Consider replacing `dqa` with something like, `data-quality`, because `dqa` may not convey the purpose of the data stream to users. (For example, is `dqa` a combination of the `d` from _data quality_, and `qa` from an abbreviation of _quality assurance_?)

expanded:

```ts
export const RESULTS_DATA_STREAM_NAME = '.kibana-data-quality-dashboard-results';

const RESULTS_INDEX_TEMPLATE_NAME = '.kibana-data-quality-dashboard-results-index-template';
const RESULTS_COMPONENT_TEMPLATE_NAME = '.kibana-data-quality-dashboard-results-mappings';
const ECS_COMPONENT_TEMPLATE_NAME = '.kibana-data-quality-dashboard-ecs-mappings';
```